cmd/bblfsh-tools/common.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"io/ioutil"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/bblfsh/sdk/protocol"
+	"github.com/bblfsh/sdk/uast"
+	"github.com/bblfsh/tools/tools"
+	errors "srcd.works/go-errors.v0"
+)
+
+var (
+	ErrorInvalidTool = errors.NewKind("invalid tool %s")
+)
+
+type Common struct {
+	Address  string `long:"address" description:"server adress to connect to" default:"localhost:9432"`
+	Language string `long:"language" description:"language of the input" default:"auto"`
```

Comment:
Remove the default from language. This was wrong on the server. To get the default language, you should just set the default value (empty string).